<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Sihai Dave Zhao</title>
    
    <link rel="stylesheet" href="css/styles.css">
  </head>

  <body>
    <div class="container">

      <div class="top_gutter"></div>
      
      <div class="navbar">
    	<div class="name"><a href="index.html">SIHAI DAVE ZHAO</a></div>
    	<div class="research"><span class="active">RESEARCH</span></div>
    	<div class="publications"><a href="publications.html">PUBLICATIONS</a></div>
    	<div class="teaching"><a href="teaching.html">TEACHING</a></div>
      </div>
      
      <div class="content">
	<div class="heading">how do we make sense of genomic data?</div>
	
	<div class="main">
      	  <img src="img/research.svg">
	  
	  I started out in the natural sciences. My main focus in college was on organic chemistry, but I was interested in everything. I eventually realized that studying statistics would let me, as Tukey said, <a href="https://nationalmedals.org/laureate/john-wilder-tukey/">play in everyoneâ€™s backyard</a>. So now I am a statistician, and I am interested in the interaction between statistics and science, particularly biology and genomics.
	  
	  <p>
	    One challenge is that biological questions are explanatory in nature, asking how and why things are the way they are; yet statistics is for the most part descriptive. I am interested in how to <a href="https://rss.onlinelibrary.wiley.com/doi/epdf/10.2307/2981525">bridge this gap</a> between description and explanation. I currently study spatial transcriptomics, and I have recently also worked on:

	    <ul>
	      <li>
		Zhu, H., Zhao, S.D., Ray, A., Zhang, Y., and Li, X. (2022).<br>
		A comprehensive temporal patterning gene network in Drosophila medulla neuroblasts revealed by single-cell RNA sequencing.<br>
		<em>Nature Communications</em>, 13, 1247.
	      </li>
	      
 	      <li>
		Avalos, A., Fang, M., Pan, H., Lluch, A. R., Lipka, A. E., Zhao, S. D., Giray, T., Robinson, G. E., Zhang, G., and Hudson, M. E. (2020).<br>
		Genomic regions influencing aggressive behavior in honey bees are defined by colony allele frequencies.<br>
		<em>Proceedings of the National Academy of Sciences</em> 117, 17135&ndash;17141.
	      </li>
	    </ul>
	  </p>
	  
	  <p>
	    Another challenge is that high-throughput genomic data allow a huge number of biological questions to be simultaneously explored, but more questions also leads to more inductive uncertainty. I am interested in the fundamental statistical principles that make it possible to reduce this uncertainty. I'm currently trying to better understand the nature of empirical Bayesian inference, which is used widely in genomics but has a bit of an <a href="https://projecteuclid.org/journals/statistical-science/volume-34/issue-2/Bayes-Oracle-Bayes-and-Empirical-Bayes/10.1214/18-STS674.full">identity crisis</a>. I also study mediation analysis models, which we have found to exhibit a somewhat surprising property that can be used to improve statistical power at the price of robustness.
	    
	    <ul>
	      <li>
		Barbehenn*, A. and Zhao, S.D.<br>
		A nonparametric regression approach to asymptotically optimal estimation of normal means.<br>
		<a href="http://export.arxiv.org/abs/2205.00336">arxiv:2205.00336</a>
	      </li>
	      
	      <li>
		Zhou*, R. R., Wang, L., and Zhao, S. D. (2020).<br>
		Estimation and inference for the indirect effect in high-dimensional linear mediation models.<br>
		<em>Biometrika</em> 107, 573&ndash;589.<br>
		(An earlier version of this paper was a winner of a 2018 American Statistical Association Section of Statistics in Genomics and Genetics distinguished student paper award.)
	      </li>
	    </ul>
	      
	  </p>

	  <p>
	    Finally, I am interested in developing new analysis procedures. My <a href="publications.html">past work includes</a> multiple hypothesis testing, precision matrix estimation, and high-dimensional survival analysis methods.
	  </p>
	  
	  <p>
	    My research has been supported by:
	    <ol>
	      <li>
		Foss (PI), Zhao (Co-PI).<br>
		A Statistical Approach to Nonlocal Compression for Supervised Learning, Semi-Supervised Learning, and Anomaly Detection.<br>
		<em>Sandia National Labs</em>, LDRD22-0599, 2021&ndash;2023
	      </li>
	      
 	      <li>
		Robinson (PI), Zhao (Co-PI).<br>
		Gut Microbiome Effects on Brain and Behavior.<br>
		<em>NSF</em> IOS-2120378, 2021&ndash;2024.
	      </li>
	      
 	      <li>
		Li (PI), Zhao (Co-PI).<br>
		Computational Reconstruction of Gene-Gene Dynamics in Temporal Patterning of Drosophila Medulla Neuroblasts from Single-Cell RNA-Seq.<br>
		<em>NSF-Simons Center for Quantitative Biology at Northwestern University</em>, 2018&ndash;2019.
	      </li>
	      
 	      <li>
		Zhao (PI).<br>
		Theory and Methods for Simultaneous Signal Analysis in Integrative Genomics.<br>
		<em>NSF</em> DMS-1613005, 2016&ndash;2019.
	      </li>
	    </ol>
	  </p>
	  
	</div>
	
    </div>

    
    
  </body>
</html>
