<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Sihai Dave Zhao</title>
    
    <link rel="stylesheet" href="css/styles.css">
  </head>

  <body>
    <div class="container">

      <div class="top_gutter"></div>
      
      <div class="navbar">
    	<div class="name"><a href="index.html">SIHAI DAVE ZHAO</a></div>
    	<div class="research"><span class="active">RESEARCH</span></div>
    	<div class="publications"><a href="publications.html">PUBLICATIONS</a></div>
    	<div class="teaching"><a href="teaching.html">TEACHING</a></div>
      </div>
      
      <div class="content">
	<div class="heading">I study statistics for genomics.</div>
	
	<div class="main">
      	  <img src="img/research.svg">

	  A central challenge of scientific research is that raw data cannot directly "enter the mind" to give understanding [<a href="#fisher">1</a>]. This difficulty is especially pronounced in genomics, where experiments can simultaneously measure data on a huge number of biological features, such as mutations or genes.
	  
	  <p>
	    I study large-scale statistical inference approaches for making sense of genomic data. I also use them to study questions in behavioral neuroscience and other substantive fields.
	  </p>

	  <p>
	    <em>Statistical research</em>
	  </p>
	  
	  <p>
	    The field of statistics has been changing rapidly. My view is that statistics is the study of how to make meaning out of data, and thus that statistics research encompasses not only (i) the development and analysis of new procedures for inferring pre-defined population-level quantities from observed data, but also (ii) the specification and definition of new population-level quantitative constructs that can carry substantive meaning [<a href="#nelder">2</a>]. 
	  </p>
	  
	  <p>
	    My statistical research program spans these activities in the context of genomics.
	    
	    <ol class="long_list">
	      <li>I introduce new quantities useful for understanding highly multiplexed genomic data. I am currently interested in single-cell and spatial transcriptomics. For example, I recently studied <a href="https://www.nature.com/articles/s41467-024-49457-w">transcript colocalization</a> for single molecule-resolution spatial transcriptomics.</li>
	      
	      <li>I create new methods to estimate and perform inference on these quantities. I mainly draw from large-scale inference procedures and high-dimensional statistics. For example, I recently studied <a href="https://academic.oup.com/biomet/article/107/3/573/5829472">high-dimensional mediation analysis</a>.</li>
	      
	      <li>I study the fundamental statistical phenomena underlying large-scale inference. I am currently interested in simultaneous estimation problems. For example, I am currently developing a new <a href="http://export.arxiv.org/abs/2205.00336">nonparametric regression-based reformulation of empirical Bayes estimation</a>.</li>
	    </ol>
	  </p>

	  <p>
	    <em>Scientific research</em>
	  </p>

	  <p>
	    I am a core member of the <a href="https://www.igb.illinois.edu/research/GNDP">Gene Networks in Neural & Developmental Plasticity</a> (GNDP) research theme at the Carl R. Woese Institute for Genomic Biology. Each theme is a tightly-organized <a href="https://www.nature.com/articles/525308a">multidisciplinary team</a> of investigators from numerous departments across campus who work together to tackle grand challenges questions typically outside the scope of an individual lab.
	  </p>

	  <p>
	    My scientific research program, in close collaboration with my GNDP colleagues, is to study the genomic basis of social behavior. I am particularly interested in how genomic information is integrated with information at higher levels of biological organization in the brain. For example, I am the PI of an NIH R01 grant to study <a href="https://reporter.nih.gov/search/vxn_ld5d7Uq1hrogW8YeaQ/project-details/11083227">multimodal network interactions underlying resiliency</a>.
	  </p>
	  
	  <p>
	    <em>Funding</em>
	  </p>
	  
	  <p>
	    My work is or has been supported by:
	    <ol class="long_list">
	      <li>
		Zhao (PI), Bonthuis (Co-I), Gritton (Co-I), Vlasov (Co-I).<br>
		CRCNS: Multimodal network interactions for internal state dynamics of resiliency.<br>
		<em>NIH</em>, R01AT013189, 2024&ndash;2028.
	      </li>
	      
	      <li>
		Han (PI), Zhao (Co-PI).<br>
		Integrated experimental and statistical tools for ultra-high-throughput spatial transcriptomics.<br>
		<em>NIH</em>, R21HG013180, 2023&ndash;2025.
	      </li>

	      <li>
		Foss (PI), Zhao (Co-PI).<br>
		A Statistical Approach to Nonlocal Compression for Supervised Learning, Semi-Supervised Learning, and Anomaly Detection.<br>
		<em>Sandia National Labs</em>, LDRD22-0599, 2021&ndash;2023.
	      </li>
	      
 	      <li>
		Robinson (PI), Zhao (Co-PI).<br>
		Gut Microbiome Effects on Brain and Behavior.<br>
		<em>NSF</em> IOS-2120378, 2021&ndash;2024.
	      </li>
	      
 	      <li>
		Li (PI), Zhao (Co-PI).<br>
		Computational Reconstruction of Gene-Gene Dynamics in Temporal Patterning of Drosophila Medulla Neuroblasts from Single-Cell RNA-Seq.<br>
		<em>NSF-Simons Center for Quantitative Biology at Northwestern University</em>, 2018&ndash;2019.
	      </li>
	      
 	      <li>
		Zhao (PI).<br>
		Theory and Methods for Simultaneous Signal Analysis in Integrative Genomics.<br>
		<em>NSF</em> DMS-1613005, 2016&ndash;2019.
	      </li>
	    </ol>
	  </p>

	  <hr>
	  
	  <p id="fisher">
	    [1]. Fisher (1922). On the mathematical foundations of theoretical statistics. <em>Philosophical Transactions of the Royal Society A</em> 222, 592&ndash;604.
	  </p>
	  <p id="nelder">
	    [2]. I think this activity lies at the heart of the emerging discipline of data science, as anticipated by Nelder: "I see statistics as giving a central place to the theory and practice of the <em>matching of theory to data</em>".<br>
	    Nelder (1986). Statistics, science and technology. <em>Journal of the Royal Statistical Society: Series A</em> 149, 109&ndash;121.
	  </p>
	  
	</div>
	
    </div>

    
    
  </body>
</html>
